src/server/utils/optimize.ts

```diff
@@ -0,0 +1,6 @@
+export const optimizeSnippet = `<style>.async-hide { opacity: 0 !important} </style>
```

Comment:
imo we should place this in the helmet `script`s instead

Reply:
We made the decision to not integrate this script as we considered the effect it has to be worse than flickering. If we want to prevent the root issue we should integrate it server-side, probably.

Reply:
Reasonable enough, can we close this then?

Reply:
Yes.